src/test/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/resource/AppointmentMappingResourceIntTest.kt

```diff
@@ -28,9 +29,10 @@ import java.time.temporal.ChronoUnit
 private const val NOMIS_EVENT_ID = 1234L
 private const val APPOINTMENT_INSTANCE_ID = 4444L
 
+@OptIn(ExperimentalCoroutinesApi::class)
```

Comment:
do you still need this opt in?

Reply:
Perhaps not, I just added it as it was an intellij compiler warning